tests/Unit/Evolution/DiscontinuousGalerkin/SlopeLimiters/Test_WenoHelpers.cpp

```diff
@@ -82,11 +97,215 @@ void test_oscillation_indicator_3d() noexcept {
   CHECK(indicator == approx(expected));
 }
 
+struct ScalarTag : db::SimpleTag {
+  using type = Scalar<DataVector>;
+  static std::string name() noexcept { return "Scalar"; }
+};
+
+template <size_t VolumeDim>
+struct VectorTag : db::SimpleTag {
+  using type = tnsr::I<DataVector, VolumeDim>;
+  static std::string name() noexcept { return "Vector"; }
+};
+
+void test_reconstruction_1d() noexcept {
+  const double neighbor_linear_weight = 0.001;
+  const auto mesh = Mesh<1>({{5}}, Spectral::Basis::Legendre,
+                            Spectral::Quadrature::GaussLobatto);
+  const auto coords = logical_coordinates(mesh);
+
+  const auto evaluate = [&coords](
```

Comment:
maybe something like `evaulate_expected_result` or something? Just `evaluate` seems comparable to naming functions `function`

Reply:
This is used to evaluate the input as well the expected result. Would you accept `evaluate_polynomial` ?